code/citadel/organs/genitals.dm

```diff
@@ -198,21 +208,28 @@
 	if(H.is_groin_exposed(worn_stuff))
 		if(H.has_penis())
 			genitals_to_add += H.getorganslot("penis")
-		if(H.has_vagina())
-			genitals_to_add += H.getorganslot("vagina")
+		else
+			if(H.has_vagina()) //Sadly, a penis would cover the vagina and they use the same layer, so don't draw both.
+				genitals_to_add += H.getorganslot("vagina")
 	if(H.is_chest_exposed(worn_stuff))
 		if(H.has_breasts())
 			genitals_to_add += H.getorganslot("breasts")
 	var/image/I
 	//start applying overlays
-	for(var/layer in relevent_layers)
+	for(var/layer in relevant_layers)
 		var/layertext = genitals_layertext(layer)
 		for(var/obj/item/organ/genital/G in genitals_to_add)
 			var/datum/sprite_accessory/S
 			switch(G.type)
 				if(/obj/item/organ/genital/penis)
 					S = cock_shapes_list[G.shape]
 					size = G.size
```

Comment:
True, this was kind of a hacky tacked-on just to make it work, before I made more things include a size in the imagename ^^.